README.md

```diff
@@ -128,6 +123,81 @@ Routemaster::Client.monitor_subscriptions
 #  } ... ]
 ```
 
+## CLI usage
+
+This gem includes the `rtm` binary, which can be used to interact with a
+Routemaster bus.
+
+### Commands
+
+    rtm token add [options] SERVICE TOKEN
+
+Adds `TOKEN` to the list of API tokens permitted to use the bus API. `SERVICE`
+is a human-readable name for this token.
+
+
+    rtm token del [options] TOKEN
+
+Removes `TOKEN` from permitted tokens if it exists.
+
+
+    rtm token list [options]
+
+Lists currently permitted API tokens.
+
+
+    rtm sub add [options] -c|--callback URL -o|--topics TOPICS [--latency MS] [--max COUNT]
+
+Adds (or updates) a subscription. Note that the `TOKEN` passed in `option` must be
+that of the subscriber, not a root token.
+
+- `URL` must be HTTPS and include an authentication username (used by the bus
+  when delivering events).
+- `TOPICS` is a comma-separated list of topic names.
+- `MS`, if specified, is the target delivery latency for this subscriber (ie.
+  how long to buffer events). 
+- `COUNT`, if specified, is the maximum number of events in a delivered batch.
+
+
+    rtm sub del [options] [-o|--topics TOPICS]
```

Comment:
When GitHub renders this as HTML this line for some reason isn't getting formatted as a code block.



![screen shot 2017-07-27 at 11 08 09](https://user-images.githubusercontent.com/306583/28665581-f8bf178c-72bb-11e7-8417-0ad71efa097c.png)